src/main/java/com/hms_networks/americas/sc/extensions/system/time/SCTimeUtils.java

```diff
@@ -39,6 +48,34 @@
   /** The designator for the local time zone. (i.e. -05:00, +02:00, Z, etc.) */
   private static String localTimeZoneDesignator = null;
 
+  /** Boolean value indicating if tag data is exported using UTC time stamps. */
+  private static boolean tagDataExportedInUtc = false;
+
+  /**
+   * Boolean value indicating of the tag data is exported using UTC time stamps option has been read
+   * from configuration.
+   */
+  private static boolean didReadTagDataExportedInUtc = false;
+
+  /**
+   * Gets a boolean value indicating if tag data is exported using UTC time stamps.
+   *
+   * @return true if tag data is exported using UTC time stamps, false otherwise
+   * @throws Exception if unable to get UTC export value
+   */
+  public static boolean getTagDataExportedInUtc() throws Exception {
```

Comment:
Should this be EWException? am i missing one?

Reply:
The system control block class has been observed to throw more than just EWException, thus the Exception class has been used. This is consistent with existing SCB method in library at https://github.com/hms-networks/sc-ewon-flexy-extensions-lib/blob/0bb80dbd8db3610ff65f60057b275922df8c6ac9/src/main/java/com/hms_networks/americas/sc/extensions/system/http/SCHttpUtility.java#L52